src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import { FC, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const [selectedTab, setSelectedTab] = useState(true);
```

Comment:
It is confusing that selectedTab is not actually a selected tab, but a boolean value
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig